app/models/user.rb

```diff
@@ -12,36 +14,54 @@ class User < ApplicationRecord
   validates :password, presence: true,
             length: {minimum: Settings.user.password}, allow_nil: true
 
+  class << self
+    def digest string
+      cost = if ActiveModel::SecurePassword.min_cost
+               BCrypt::Engine::MIN_COST
+             else
+               BCrypt::Engine.cost
+             end
```

Comment:
Đoạn này xử lý ngắn gọn lại được ko em

Reply:
Có thể viết là cost = ActiveModel::SecurePassword.min_cost ? BCrypt::Engine::MIN_COST : BCrypt::Engine.cost ạ nhưng viết dài quá nên em đổi thành ntn ạ

Reply:
```suggestion
      min_cost = ActiveModel::SecurePassword.min_cost
      cost = min_cost ? BCrypt::Engine::MIN_COST : BCrypt::Engine.cost
```
Viết như này thì có chạy không em

Reply:
Hic em update lai a